apps/trading/stores/global.ts

```diff
@@ -14,6 +16,8 @@ interface PageTitleStore {
 
 export const useGlobalStore = create<GlobalStore>()((set) => ({
   marketId: LocalStorage.getItem('marketId') || null,
+  onBoardingDismissed: false,
```

Comment:
this acts like temporary dismiss for getting user to next steps like deposit or order. If we change it to the persisted store, it will not distinguish anymore from the second one type of dismissal, which happens when user eventually reach the whole onboarding. So let's keep it this way.